modules/core/src/passes/pick-layers-pass.js

```diff
@@ -10,8 +10,10 @@ const PICKING_PARAMETERS = {
 export default class PickLayersPass extends LayersPass {
   render(props) {
     if (props.pickingFBO) {
+      this.toScreen = false;
```

Comment:
Nits:
- A comment perhaps? Or slightly more focused naming? 
- Maybe initialize field to false in constructor (upcoming typescript prep)